druntime/src/importc.h

```diff
@@ -134,6 +134,7 @@ typedef unsigned long long __uint64_t;
 #define __ptr64
 #define __unaligned
 #define _NO_CRT_STDIO_INLINE 1
+#define _stdcall __stdcall
```

Comment:
this is the only line that should be part of this PR. Please revert all the other whitespace changes

Reply:
yes i will do. it is a stupid extension of my vscode